api/apps/api/src/modules/legacy-project-import/application/run-legacy-project-import.command.ts

```diff
@@ -0,0 +1,23 @@
+import { Command } from '@nestjs-architects/typed-cqrs';
+import { Either } from 'fp-ts/lib/Either';
+import { GenerateLegacyProjectImportPiecesErrors } from '../domain/legacy-project-import/legacy-project-import';
+import {
+  LegacyProjectImportRepositoryFindErrors,
+  LegacyProjectImportRepositorySaveErrors,
+} from '../domain/legacy-project-import/legacy-project-import.repository';
+
+export type RunLegacyProjectImportError =
+  | GenerateLegacyProjectImportPiecesErrors
+  | LegacyProjectImportRepositorySaveErrors
+  | LegacyProjectImportRepositoryFindErrors;
+
+export type RunLegacyProjectImportResponse = Either<
+  RunLegacyProjectImportError,
+  true
+>;
+
+export class RunLegacyProjectImport extends Command<RunLegacyProjectImportResponse> {
+  constructor(public readonly projectId: string) {
```

Comment:
I would use `ResourceId` here for the sake of consistency with another legacy project import commands: [`CompleteLegacyProjectImportPiece`](https://github.com/Vizzuality/marxan-cloud/blob/a201da84896df5dd222c2c6d901b81364f0fe917/api/apps/api/src/modules/legacy-project-import/application/complete-import-legacy-project-piece.command.ts#L7), [`MarkLegacyProjectImportPieceAsFailed`](https://github.com/Vizzuality/marxan-cloud/blob/a201da84896df5dd222c2c6d901b81364f0fe917/api/apps/api/src/modules/legacy-project-import/application/mark-legacy-project-import-piece-as-failed.command.ts#L7) and [`MarkLegacyProjectImportAsFailed`](https://github.com/Vizzuality/marxan-cloud/blob/46c10a2d6415a00197d88f71bcc06bd278b8f5c0/api/apps/api/src/modules/legacy-project-import/application/mark-legacy-project-import-as-failed.command.ts#L6)

```suggestion
  constructor(public readonly projectId: ResourceId) {
```